tests/compile-fail/borrow_box.rs

```diff
@@ -0,0 +1,14 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#![deny(clippy)]
+#![allow(boxed_local)]
+#![allow(blacklisted_name)]
+
+pub fn test(foo: &Box<bool>) { //~ ERROR you seem to be trying to use `&Box<T>`
```

Comment:
uhm... `let foo: &T = some_extern_crate::bar()` will work just fine, even if that function returns a `&Box<T>`: https://is.gd/kdISz8